.gitignore

```diff
@@ -46,3 +46,6 @@ yarn-error.log
 
 # Ignore package-lock.json because we use yarn
 package-lock.json
+
+#Jetbrains Tools
+.idea/
```

Comment:
Thanks for all the suggestions. It appears that the .eslint rules are not set. Even when I configured the IDE to use that .eslint file, it didn't work as expected. I think that file should handle all these rules instead of manually keeping track of it.

Reply:
Cool, thanks to you!